okapi/services/ocpl/paths/gplog_entries.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+# TODO:...
+# This method is the simplest of all. It just returns all cashes, in any order.
```

Comment:
> mzylowski 8 hours ago Member
caches :)

@mzylowski, proszę cię :) - ja tam robię copy-paste na maxa i to nie jest gotowe rozwiązanie tylko pierwszy wstępny rzut na branch implementacyjny